appeals/api/src/database/seed/data-test.js

```diff
@@ -114,6 +114,7 @@ const appealFactory = ({
 		},
 		applicationReference: generateLpaReference(),
 		address: { create: formattedAddress },
+		isGreenBelt: true,
```

Comment:
Should this be passed as a parameter or randomised, to ensure some test appeals have it set to true and others false?